drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -311,6 +311,10 @@ def age(date = Date.current)
     GrdaWarehouse::Hud::Client.age(date: date, dob: self.DOB)
   end
 
+  def veteran?
+    self.VeteranStatus == 1
+  end
+
```

Comment:
Not a big deal, but this could go [into the shared concern](https://github.com/greenriver/hmis-warehouse/blob/release-101/app/models/concerns/hud_concerns/client.rb) and be removed from the grda warehouse [class](https://github.com/greenriver/hmis-warehouse/blob/eca466263a61e6d08703d1aefe50dae2f501d68c/app/models/grda_warehouse/hud/client.rb#L1985-L1987). The intention, at least originally, was to move shared logic to the concern over time.